app/models/miq_report/generator.rb

```diff
@@ -314,6 +313,7 @@ def generate_basic_results(options = {})
       :filter           => conditions,
       :include_for_find => get_include_for_find,
       :where_clause     => where_clause,
+      :conditions       => options[:where_clause],
```

Comment:
@kbrock, help me understand this.  Before, we were merging the existing `self.where_clause` and the `options[:where_clause]` into the `options[:where_clause]`.  Now, we don't merge them together but merge `where_clause` into `options[:where_clause]` and tack/merge on `options[:conditions]` with the `options[:where_clause]`

So, my questions...
1) What forced us to do this in the first place?  Did the UI use the where_clause and not the conditions?
2) Is there any difference in behavior now?  What does this buy us besides removing the arel internals?

I like the removal but don't understand how it used to work vs. now.

Reply:
@jrafanie I'll lookup what parameters are passed into there. There sure are a bunch of paths that pass `options` into reports. Good call.

I have been removing sql munging since at least October 2015.
This looks like it has been removable for at least a year.